sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.institution.controller.getOverawardsByStudent.e2e-spec.ts

```diff
@@ -0,0 +1,108 @@
+  createFakeUser,
+  createFakeStudentAssessment,
+  createFakeApplication,
+} from "@sims/test-utils";
+import { Repository } from "typeorm";
+import {
+  Application,
+  AssessmentTriggerType,
+  DisbursementOveraward,
+  DisbursementOverawardOriginType,
+  Student,
+  StudentAssessment,
+  User,
+} from "@sims/sims-db";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import { OverawardDetailsAPIOutDTO } from "../../models/overaward.dto";
+import { getUserFullName } from "../../../../utilities";
+
+describe("OverawardInstitutionController(e2e)-getOverawardsByStudent", () => {
+  let app: INestApplication;
+  let userRepo: Repository<User>;
+  let studentRepo: Repository<Student>;
+  let assessmentRepo: Repository<StudentAssessment>;
+  let applicationRepo: Repository<Application>;
+  let disbursementOverawardRepo: Repository<DisbursementOveraward>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    userRepo = dataSource.getRepository(User);
+    studentRepo = dataSource.getRepository(Student);
+    assessmentRepo = dataSource.getRepository(StudentAssessment);
+    applicationRepo = dataSource.getRepository(Application);
+    disbursementOverawardRepo = dataSource.getRepository(DisbursementOveraward);
+  });
+
+  it("Should return student overawards", async () => {
+    // Arrange
+    const user = await userRepo.save(createFakeUser());
+    const student = await studentRepo.save(createFakeStudent());
+    // Prepare the student assessment to create overaward.
+    const application = await applicationRepo.save(
+      createFakeApplication({
+        student,
+      }),
+    );
+    const studentAssessment = await assessmentRepo.save(
+      createFakeStudentAssessment({ auditUser: user, application }),
+    );
+    application.currentAssessment = studentAssessment;
+    await applicationRepo.save(application);
+    // Create an overaward.
+    const reassessmentOveraward = createFakeDisbursementOveraward({ student });
+    reassessmentOveraward.studentAssessment = studentAssessment;
+    reassessmentOveraward.creator = user;
+    reassessmentOveraward.disbursementValueCode = "CSLF";
+    reassessmentOveraward.overawardValue = 500;
+    reassessmentOveraward.originType =
+      DisbursementOverawardOriginType.ReassessmentOveraward;
+    reassessmentOveraward.addedBy = user;
+    reassessmentOveraward.addedDate = new Date();
+    await disbursementOverawardRepo.save(reassessmentOveraward);
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    const endpoint = `/institutions/overaward/student/${student.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .then((response) => {
```

Comment:
Is it possible to follow what we have been doing more lately about validating the entire body response instead of the individual properties?